### ISSUE TITLE
Use your bundle

### ISSUE THREAD

**Issue by disjfa** [toiba/FullCalendarBundle#2]
Hey, i was planning on making a bundle for the fullcalendar. Saw your bundle (also saw you forked it). I was planning to add some abstract classes and traits to "force" people on easy extending entities and items.

I have a couple of options, fork and "steal" event listeners and useful stuff, add some to the bundle and make it better. The latter seems more awesome in my view. So there is my question, what is your opinion in this.

I just want to let you know i have not checked or thought of how to get things started. But this would not be my first full calendar setup. But as far as i know, more people is more awesome. But i want to integrate it into my https://github.com/disjfa/glynn-admin-symfony4 app or/and be able to integrate into a skeleton app.

**Comment by tattali** [comment]
Hi, this bundle is fully compatible with symfony 4, so you can install it and be free to add your stuff.
But i don't understand what do you want add.

**Comment by disjfa** [comment]
Quick example

https://github.com/disjfa/FullCalendarBundle
https://github.com/disjfa/event-bundle

This is a realy realy quick example, to keep it simple. I added an interface for the events. Added an orm trait so that can be implemented and added a basic form. Added a custom bundle, maybe for show or to use for me, to show what can be done.

This does add some forced bundles people could not use. But it is probably the way people are gonna use it.

But it makes it easier to implement and extend (at least the interface). Or just to easily use traits for entities.
